uint8 ask_for_trees
---
sensor_msgs/Image[] trees
